stdeb.cfg

```diff
@@ -0,0 +1,7 @@
+[DEFAULT]
+Package: python-h5py      
+XS-Python-Version: >= 2.6
+Maintainer: Jerome Kieffer <jerome.kieffer@esrf.fr>
+Build-Depends: python-dev
+Recommends: python-numpy 
```

Comment:
This doesn't seem related to the rest of the PR. And I'd rather leave Debian packaging out of the repo entirely.